addons/mail/static/src/models/emoji_view.js

```diff
@@ -54,5 +54,10 @@ registerModel({
         emojiGridViewAsHovered: one('EmojiGridView', {
             inverse: 'hoveredEmojiView',
         }),
+        popoverViewTopOwner: one('PopoverView', {
+            compute() {
+                return this.emojiGridItemViewOwner.emojiGridRowViewOwner.emojiGridViewOwner.emojiPickerViewOwner.popoverViewOwner;
+            }
+        })
```

Comment:
I feel that it's best to compute the `EmojiPickerView`, and keep `.popoverViewOwner` in the branching.
Just because `PopoverView` is a container of emoji picker. Right now it's just `PopoverView`, but there's desire to have Emoji Picker in different "container", like `Third` in one of your task!